src/css/page_sections/Menu/_Menu.scss

```diff
@@ -0,0 +1,71 @@
+.coa-Menu {
+  @media screen and (min-width: $coa-large-screen) {
```

Comment:
I've been sticking to min-width mixin
`@include media($coa-medium-screen) { ... }`
I think we should stick to either min-width or max-width because layouts can get weirdly conflicted otherwise with components shifting at different spots.
We're probably interpreting Celine's breakpoints in mocks differently as well.
This is mostly a talking point for us to discuss and address probably post usability test.

Reply:
That's a good call. I refactored Menu.scss and I'll make an issue to come back to the other three scss files in `css/page_sections/Menu`